pkg/pb/notifications.proto

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package notifications;
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+message Notification{
+    enum Scope {
+        CUSTOM = 0;
+        AUDIT = 1;
+        UPTIME = 2;
+        REPAIR = 3;
+        DISQUALIFICATION = 4;
+        GRACEFUL_EXIT = 5;
+        VETTING = 6;
+    }
+
+    enum Level {
+        INFO = 0;
+        WARN = 1;
+        ERROR = 2;
+    }
+
+    Scope scope = 1;
+    Level level = 2;
+    repeated string tags = 3;
+    string message = 4;
+    google.protobuf.Timestamp timestamp = 5 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+}
+
+message ReportRequest{
```

Comment:
How is this being used? Why do we need to send them all of this information?

Reply:
Thats what i asked myself as well. This feels way worse, than sending two notifications.

Reply:
that is something what @jtolds wanted it to send on a regular basis

Reply:
here are some requirements for audit check/uptime check notices:
1) the snoboard will need to be able to aggregate and display time series data over customizable windows. "how many audits did i fail out of the total over the last hour or the last day?" is question SNOs should be able to have the snoboard tell them.
2) the snoboard will in the long term ultimately be localized with language choices - for most message types, the snoboard should be able to display the information in the sno's local language - ergo, the satellite should not pick the actual message.
3) we want to reduce notification traffic as much as possible, so this type of notice should be batchable. we should not have to send a message for every audit fail/success. we should be able to send a message for a rollup over some period of time.